doc/reference/networking/net_offload.rst

```diff
@@ -16,3 +16,23 @@ API Reference
 
 .. doxygengroup:: net_offload
    :project: Zephyr
+
+Socket Offloading
+#################
+
+Overview
+********
+
+Beside of the network offloading API, Zephyr allows to offload networking
+functionality at the socket API level. With this approach, vendors that provide
+alternate implementation of the networking stack, exposing socket API for their
+networking devices, can easily integrate them with Zephyr.
```

Comment:
", vendors who provide an alternate implementation of the networking stack, "

", can easily integrate it with Zephyr".